tests/test_exprs.py

```diff
@@ -865,41 +865,44 @@ def test_similarity2(self, img_tbl: catalog.Table) -> None:
             _ = t[t.img.nearest('musical instrument')].show(10)
 
     def test_ids(
-            self, test_tbl: catalog.Table, test_tbl_exprs: list[exprs.Expr],
-            img_tbl: catalog.Table, img_tbl_exprs: list[exprs.Expr]
+            self, test_tbl_exprs: list[exprs.Expr], img_tbl_exprs: list[exprs.Expr],
+            sim_exprs_same_idx: list[exprs.Expr], sim_exprs_diff_idx: list[exprs.Expr]
     ) -> None:
         skip_test_if_not_installed('transformers')
         d: dict[int, exprs.Expr] = {}
-        for e in test_tbl_exprs:
+        for e in test_tbl_exprs + img_tbl_exprs + sim_exprs_diff_idx:
             assert e.id is not None
             d[e.id] = e
-        for e in img_tbl_exprs:
+        assert len(d) == len(test_tbl_exprs) + len(img_tbl_exprs) + len(sim_exprs_diff_idx)
+        nexprs_before = len(d)
+        for e in sim_exprs_same_idx:
             assert e.id is not None
             d[e.id] = e
-        assert len(d) == len(test_tbl_exprs) + len(img_tbl_exprs)
+        assert len(d) == nexprs_before + 1
```

Comment:
you're testing something here that wasn't part of the original test (its intention), which is that the two similarity exprs are identical. that should go in a different test case in test_index.

Reply:
that's a good idea. since it is focused on the expression, i moved it to test_similarity inside test_expr.py. hope you agree.
i initially thought it was ok to extend this test case as it was already verifying each id was unique. but its better separate, thanks!
<!-- Reviewable comment -OETQx0c07zCf0rLAV1Q:byeszxc -->
<!-- Sent from Reviewable.io -->